ros2/nav2/pose_publisher/pose_publisher/pose_publisher.py

```diff
@@ -0,0 +1,68 @@
+import rclpy
+from rclpy.node import Node
+from geometry_msgs.msg import PoseStamped
+import tf2_ros
+import tf2_geometry_msgs
+from tf2_msgs.msg import TFMessage
+
+
+class PosePublisher(Node):
```

Comment:
How is this node being run? Manually? If this is being run together with the navigation stack, then it would make sense to add a launch file